src/main/java/org/jsmart/zerocode/core/kafka/KafkaConstants.java

```diff
@@ -1,25 +1,30 @@
 package org.jsmart.zerocode.core.kafka;
 
 public interface KafkaConstants {
-	public static String KAFKA_BROKERS = "192.168.99.1:9092";
```

Comment:
Why the Kafka broker is a constant of some IP rather localhost?

Reply:
good spot, removed that now. Fixed.